goal_src/jak2/levels/common/airlock.gc

```diff
@@ -200,7 +205,7 @@
                                     )
                                 (and (eval!
                                        (new 'stack 'script-context (the-as basic (process->ppointer self)) self (the-as vector #f))
-                                       (the-as structure (-> obj open-test))
+                                       (the-as pair (-> obj open-test))
```

Comment:
I wonder if we should make this `eval!` function just take an `object` here - I kinda think it can be either a pair or a symbol.